.github/workflows/ci.yml

```diff
@@ -94,11 +88,10 @@ jobs:
         env:
           RAILS_ENV: test
           DATABASE_URL: postgres://postgres:postgres@localhost:5432
-          # REDIS_URL: redis://localhost:6379/0
         run: |
           bin/rails db:create
           bin/rails db:schema:load
-          bin/rails test:all
```

Comment:
Until these are stable (follow up PRs coming), no reason to be running them as they are false negatives.  Will reintroduce with more stability in the future.